openedx/features/content_tagging/api.py

```diff
@@ -0,0 +1,106 @@
+"""
+Content Tagging APIs
+"""
+from typing import List
+
+import openedx_tagging.core.tagging.api as oel_tagging
+from django.db.models import QuerySet
+from openedx_tagging.core.tagging.models import ObjectTag, Tag, Taxonomy
+from organizations.models import Organization
+
+from .models import ContentTaxonomy
+
+
+def create_taxonomy(
+    name,
+    org_owners: List[Organization] = None,
+    description=None,
+    enabled=True,
+    required=False,
+    allow_multiple=False,
+    allow_free_text=False,
+) -> ContentTaxonomy:
+    """
+    Creates, saves, and returns a new ContentTaxonomy with the given attributes.
+
+    If `org_owners` is empty/None, then the returned taxonomy is enabled for all organizations.
+    """
+    taxonomy = ContentTaxonomy.objects.create(
+        name=name,
+        description=description,
+        enabled=enabled,
+        required=required,
+        allow_multiple=allow_multiple,
+        allow_free_text=allow_free_text,
+    )
+    if org_owners:
+        set_taxonomy_org_owners(taxonomy, org_owners)
+    return taxonomy
+
+
+def set_taxonomy_org_owners(
+    taxonomy: ContentTaxonomy,
+    org_owners: List[Organization] = None,
+) -> ContentTaxonomy:
+    """
+    Updates the list of org_owners on the given taxonomy.
+
+    If `org_owners` is empty/None, then the returned taxonomy is enabled for all organizations.
+    """
+    if org_owners:
+        taxonomy.org_owners.set(org_owners)
+    else:
+        taxonomy.org_owners.clear()
+    return taxonomy
+
+
+def get_taxonomies(org_owner: Organization = None, enabled=True) -> QuerySet:
+    """
+    Returns a queryset containing the enabled taxonomies owned by the given org, sorted by name.
+
+    If you want the disabled taxonomies, pass enabled=False.
+    If you want all taxonomies (both enabled and disabled), pass enabled=None.
+    """
+    return (
+        ContentTaxonomy.objects.filter_enabled(
+            org_owner,
+            enabled,
+        )
+        .order_by("name", "id")
+        .select_subclasses()
+    )
+
+
+# Expose the oel_tagging APIs
```

Comment:
Is there a benefit to this method of importing and re-defining all the methods like this? Can't we just say `from oel_tagging import *` or use something like `tagging_api = oel_tagging` and then have people call `tagging_api.get_tags()` ?

Reply:
Ah yes, I can be much more concise here, see d0c446b269a4b9a747e6613a39c34bf5aaa1cab9